src/main/java/twitterapp/src/resources/TwitterAppResource.java

```diff
@@ -85,6 +85,28 @@ public Response postTweet(RequestBody input) throws Exception{
         return Response.ok(twitterPost.get(), MediaType.APPLICATION_JSON_TYPE).build();
     }
 
+    @POST
+    @Path("/tweet/reply")
+    @Consumes("application/json")
+    public Response replyTweet(RequestBody input) throws Exception{
```

Comment:
i'd recommend to create a separate request for the endpoint
ReplyTweetRequest and respective classes for other endpoints, this allows the API contract to be more explicit